entrypoint.sh

```diff
@@ -0,0 +1 @@
+poetry run gunicorn --bind "0.0.0.0:${PORT:-5000}" 'todo_app.app:create_app()'
```

Comment:
Don't forget to add a shebang to the top of your script files (like you have with the deploy script!)